geode-for-redis/src/main/java/org/apache/geode/redis/internal/statistics/GeodeRedisStats.java

```diff
@@ -30,170 +32,197 @@
 import org.apache.geode.redis.internal.commands.RedisCommandType;
 
 public class GeodeRedisStats {
+
+  public static final String STATS_BASENAME = "GeodeForRedisStats";
+  private static final String GENERAL_CATEGORY = "General";
+
   @Immutable
-  private static final StatisticsType type;
+  private static final Map<String, StatisticsType> statisticTypes = new HashMap<>();
```

Comment:
Yes, I wanted to, but since I also have this "General" category (which only exists to satisfy the stat naming and isn't a real Category) I opted to use String as the key. Perhaps you have a suggestion on how to do this better?

Reply:
It seems like you should either add General as a Category (does that cause other problems?) or keep the general one out of these maps. If you do the latter then you would just have a map for the category ones and you would have another instance variable that just directly references the general one. So in this example you would have "EnumMap<Category, StatisticsType> categoryTypes;" and "StatisticsType generalType".

Reply:
I opted to keep "General" separate and switch to an `EnumMap`.